### PR TITLE
ADDED:  Login Working | TODO CSS | TODO ERROR

### PR DESCRIPTION
**Pull request by ShutUpChips15**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+import { MatCardModule } from '@angular/material/card';
+import { MatInputModule } from '@angular/material/input';
+import { MatButtonModule } from '@angular/material/button';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { LoginComponent } from './login/login.component';
 import { StudentsModule } from './students/students.module';
 import { TeacherModule } from './teacher/teacher.module';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
```
Comment (jasonncleveland) [inline src/app/app.module.ts:14]: Are these two imports necessary? If so, what are they for.

File: src/app/app.module.ts [hunk 2]
```diff
@@ -14,9 +22,16 @@ import { TeacherModule } from './teacher/teacher.module';
...
+    FormsModule,
+    ReactiveFormsModule,
+    MatFormFieldModule,
+    MatCardModule,
+    MatInputModule,
+    MatButtonModule,
     StudentsModule,
     TeacherModule,
-    AppRoutingModule
+    AppRoutingModule,
```
Comment (jasonncleveland) [inline src/app/app.module.ts:33]: See above

File: src/app/login/login.component.ts [hunk 3]
```diff
@@ -7,9 +9,42 @@ import { Component, OnInit } from '@angular/core';
...
+        
+      }
+      else if (username === "student"){
+        console.log("Yes: Student");
+        this.router.navigateByUrl('/students')
+        
+      }
+    }
+    else {
+      console.log("Unvalid");
```
Comment (jasonncleveland) [inline src/app/login/login.component.ts]: Should say "invalid". Unvalid is not a word

**Review by jasonncleveland — CHANGES_REQUESTED** [review]

File: src/app/login/login.component.ts (hunk 3 above)
Comment (ShutUpChips15) [reply src/app/login/login.component.ts]: Josh's English.

File: src/app/app.module.ts (hunk 1 above)
Comment (ShutUpChips15) [reply src/app/app.module.ts:14]: added by ng.

File: src/app/app.module.ts (hunk 2 above)
Comment (ShutUpChips15) [reply src/app/app.module.ts:33]: added by ng.

**Review by jasonncleveland — APPROVED** [review]

**Pull request merged**